_posts/2020-11-16-biohackathon.md

```diff
@@ -0,0 +1,51 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+tags: [training, hackathon]
+title: Biohackathon Europe 2020
+supporters: elixir
+location: Barcelona (online)
+---
+
+
+Last week we had the [__BioHackathon Europe 2020__](https://www.biohackathon-europe.org/) - online for the first time and with participants from all over the world and the Galaxy community has participated in several [projects](https://www.biohackathon-europe.org/projects.html):
+
+
+### [CNV detection software containerisation and benchmark](https://github.com/elixir-europe/BioHackathon-projects-2020/tree/master/projects/7)
```

Comment:
@wm75 would be nice to have your input here... not sure if everything that I'm saying is correct :) Thanks!